sql/src/test/java/io/crate/analyze/CopyAnalyzerTest.java

```diff
@@ -115,6 +116,29 @@ public void testCopyFromParameter() throws Exception {
         assertThat(BytesRefs.toString(value), is(path));
     }
 
+    @Test
+    public void testCopyFromWithInputFormatJson() {
```

Comment:
It would be nice to either state more explicitly in the method name what aspect a test is testing or reduce the assertions to the relevant parts.

The tests below all assert the `table` and `uri` but I suspect that this is not relevant - but only the format handling.

If it is not clear what aspect a test is testing it is incredible hard to judge later on if they're still required after a refactoring.

Reply:
Fair point. I've removed the unnecessary assertions. 

I prefer to use the naming style that I used in other tests (i.e. `methodToTest_givenXXXX_thenXXXX`), unless you have another preference? I used the other approach originally to be consistent with the current naming style in this class, so perhaps these could be refactored in another pr.